styles/styles.css

```diff
@@ -0,0 +1,155 @@
+header {
+  background-image: url('../assets/hero.jpg');
+  /*background-position: 50% 50%;*/
+  background-size: cover;
+  background-repeat: no-repeat;
+  border-bottom: solid;
+  border-color: #9D9494;
+  height: 100px;
```

Comment:
Pixels are standard and fine, but I encourage you to play around with em and rem (or percentages) for margins/height/etc! It'll help so everything doesn't break when you resize windows.

Edit: I saw you used them a bit more below with font size and margins! Keep exploring! 👍